src/pyimagej.egg-info/PKG-INFO

```diff
@@ -0,0 +1,126 @@
+Metadata-Version: 2.1
```

Comment:
I think you committed the `src/pyimagej.egg-info` directory by accident, no? Can you please purge it from the history of this PR?

Reply:
:vomiting_face:

It shouldn't be part of the commit history anymore, with 616634c